src/templates/ticket.html.j2

```diff
@@ -0,0 +1,150 @@
+      color: #006ef5;
+    }
+    .selectize-control .item.active a {
+      color: #303030;
+    }
+    .selectize-control .optgroup-header {
+      display: block;
+      font-size: 1.2em;
+      color: #303030;
+    }
+</style>
+
+
+<script>
+
+$(function(){
+
+  grecaptcha.ready(function() {
+    grecaptcha.execute('6LdJl44UAAAAAGaTKjXWjhPJ5MvyXCAYGxd2m4TS', {action: 'ticket'}).then(function(token) {
+         $("#g-recaptcha-response").attr("value", token);
+      });
+  });
+
+  var $select = $('#resource-list').selectize({
+          maxItems: null,
+          valueField: 'name',
+          labelField: 'name',
+          searchField: ['name', 'facility'],
+          options: [],
+          create: false,
+          optgroupField: 'facility',
+          optgroupLabelField: 'facility',
+          optgroupValueField: 'facility',
+          render: {
+            option: function(data, escape) {
+              return '<div class="option">' +
+                  '<span class="name">' + escape(data.name) + '</span>' +
+                  '<span class="facility">' + escape(data.type) + '</span>' +
+                '</div>';
+            },
+            item: function(data, escape) {
+              return '<div class="item">' + escape(data.name) + '</a></div>';
+            },
+            optgroup_header: function(data, escape) {
+              return '<div class="optgroup-header">' + escape(data.facility) + '</div>';
+            }
+        }});
+
+  var control = $select[0].selectize;
+  // Everything goes here
+  // Parse the XML
+  function parseResources(data) {
+    // The returned data should be XML
+    $(data).find("ResourceGroup").each(function(index, rg) {
+      var facility = $(rg).find("Facility").find("Name").first();
+      console.log(facility[0].innerHTML);
+      var facility_name = facility[0].innerHTML;
+      control.addOptionGroup(facility_name, {
+        facility: facility_name
+      })
+      var resource_list = $(rg).find("Resource");
+      for (var i = 0; i < resource_list.length; i++) {
+        var resource_name = $(resource_list[i]).find("Name")[0].innerHTML;
+        var resource_id = $(resource_list[i]).find("ID")[0].innerHTML;
+        var services = $(resource_list[i]).find("Services")[0];
+        var service = $(services).find("Service")[0]
+        var type = $(service).find("Name")[0].innerHTML;
+        //console.log(resource_name)
+        control.addOption({
+            id: resource_id,
+            name: resource_name,
+            facility: facility_name,
+            type: type
+          });
+      }
+      console.log(rg);
+    });
+
+  }
+
+  // Load the resource
+  function loadResources() {
+    $.get("/rgsummary/xml", parseResources);
+  }
+
+  loadResources();
+
+});
+
+
+</script>
+
+
+
+{% endblock %}
+
+{% block content %}
+<div class="container">
+
+  <form action="/submitTicket" method="POST">
+    <div class="form-group">
+      <label for="resource-list">Resource</label>
```

Comment:
> Pick OSG Resource(s):

Reply:
That "Pick Resource(s)" is the placeholder text in the search box.  Should it be repeated?

Reply:
Ah, I missed that. `Resource(s) is fine`